src/hooks/useWalletConnection.ts

```diff
@@ -142,7 +156,7 @@ export const useWalletConnection = () => {
           }
 
           if (!isConnectedGraz) {
-            await connectGraz({
+            connectGraz({
```

Comment:
curious why this line was changed.

Reply:
i had a lint warning that the await did nothing in this case, but can add it back for now. you may want to take a look at that, not 100% sure

Reply:
Oh you're right, I'll look into it